igor-web/src/main/groovy/com/netflix/spinnaker/igor/jenkins/service/JenkinsService.groovy

```diff
@@ -103,6 +103,12 @@ class JenkinsService implements BuildService{
         return getBuild(jobName, buildNumber).genericBuild(jobName)
     }
 
+    @Override
+    int triggerBuildWithParameters(String job, Map<String, String> queryParameters) {
+        Response response = buildWithParameters(job, queryParameters)
+        return response.body.toString().toInteger()
```

Comment:
This was wrong, I have copied the code from the BuildController to extract the queued build. This method is not called as it is now, but I want to refactor so that the trigger build part in BuildController does not care whether it is jenkins or travis its triggering.
I tested some local triggering to see what the responses from igor is when triggering:
```
me@host $ curl http://172.31.0.134:8088/masters/spinnaker-jenkins/jobs/integration-tests -X PUT
168
me@host $ curl http://172.31.0.134:8088/masters/spinnaker-jenkins/jobs/whatsup?INTEGRATION_TEST_TARGET=4 -X PUT
169
```